dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/JdbcEventAnalyticsTableManager.java

```diff
@@ -115,6 +117,7 @@ public JdbcEventAnalyticsTableManager( IdentifiableObjectManager idObjectManager
         new AnalyticsTableColumn( quote( "completeddate" ), TIMESTAMP, "psi.completeddate" ),
         new AnalyticsTableColumn( quote( "created" ), TIMESTAMP, "psi.created" ),
         new AnalyticsTableColumn( quote( "lastupdated" ), TIMESTAMP, "psi.lastupdated" ),
+        new AnalyticsTableColumn( quote( STORED_BY_COL_NAME ), CHARACTER_50, "psi.storedby" ),
```

Comment:
there is a bit different definition for column psi.storedby (programstageinstance table) in 2.37 database: 
storedby character varying(255) COLLATE pg_catalog."default".  Make it sence to use CHARACTER_255?

Reply:
Good catch. I'll change this to `VARCHAR(255)`